base_geoengine/README.rst

```diff
@@ -24,14 +24,22 @@ The module also requires two additional python libs:
 
 for a complete documentation please refer to the `public documenation <http://oca.github.io/geospatial/index.html>`_
 
+Known issues / Roadmap
+======================
+
+ * Google layers have been removed as it was not working anyway.
```

Comment:
@yvaucher you can also remove https://github.com/yvaucher/geospatial/blob/ec85e432cf1f078860c4a8d758d2324af61b16ab/base_geoengine/data/geo_raster_layer_type.xml